h2o-core/src/main/java/water/api/RequestServer.java

```diff
@@ -314,12 +314,18 @@ public void doGeneric(String method, HttpServletRequest request, HttpServletResp
 
       resp.writeTo(response.getOutputStream());
 
+    } catch (Model.UnsupportedConversionToJavaException e) {  // This error type should be passed to clients.
```

Comment:
I still believe there should not be any special handling of just this particular state.

Reply:
Removed entire catch, so only the original with `IOException` remains